GVFS/GVFS.Platform.Mac/MacDaemonManager.cs

```diff
@@ -0,0 +1,79 @@
+using GVFS.Common;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GVFS.Platform.Mac
+{
+    /// <summary>
+    /// Class to query the configured services on macOS
+    /// </summary>
+    public class MacDaemonManager

```

Comment:
`Manager` doesn't seem like the right term here since it is only for getting a list of the daemons.  Also don't think we need `Mac` in the name since it could possibly be used for Linux?  What about `DaemonList`?

Reply:
`MacDaemonLister`?

> Also don't think we need Mac in the name

This implementation is specific to macOS (I think `launchctl` is more of a macOS artifact than Linux).